protocol/protobuf/communities.proto

```diff
@@ -10,7 +10,6 @@ package protobuf;
 message Grant {
   bytes community_id = 1;
   bytes member_id = 2;
-  string chat_id = 3;
```

Comment:
I think it would be good to keep it and leave it empty for community-level grants. In the future, we may need to prove we are members of a given channel, and the chat_id would be useful in that case.

Reply:
sure, i'll revert it